doc/examples.md

```diff
@@ -1,6 +1,6 @@
 # Example Configs
 
-Each of these examples is written in version 2.2.0 of the config. Ensure that any configuration is compatible with the version that Ignition accepts. Compatibility requires the major versions to match and the spec version be less than or equal to the version Ignition accepts.
+Each of these examples is written in version 3.0.0 of the config. Ensure that any configuration is compatible with the version that Ignition accepts. Compatibility requires the major versions to match and the spec version be less than or equal to the version Ignition accepts and greater than or equal to 3.0.0. For examples using the 2.x.y specification, see this document in the `spec2x` branch of this repository.
```

Comment:
> Compatibility requires the major versions to match and the spec version

But we're talking about the major version of the spec already here, right? Maybe something like:

> Compatibility requires the major versions to match and the minor and patch versions be less than or equal to ...

Reply:
Thinking about this more, its somewhat confusing since there's many different types of compatibility. I'm going to just drop this in favor of simpler text. If we want to talk about config compat and versioning we can put that in a separate doc or in the operator notes.